### PR TITLE
ANC Logic - diagnosis Malaria

### PR DESCRIPTION
**Pull request by anvmn**
#136

### CODE REVIEW

**Comment by anvmn** [comment]
@mariano-dagostino please review.
Note that `@todo`s that appear throughout are implemented at #138, and those for 
 `NextStepsMedicationDistribution` at `Pages.PrenatalRecurrentActivity ` will be implemented at upcomming PR for #141.

File: client/src/elm/Pages/PrenatalActivity/View.elm [hunk 1]
```diff
@@ -1480,41 +1481,35 @@ viewNextStepsContent language currentDate isChw assembled data =
...
-                            ( "next-steps-newborn-enrolment"
-                            , isJust assembled.participant.newborn
-                            )
+                            "next-steps-newborn-enrolment"
 
                         NextStepsMedicationDistribution ->
-                            ( "next-steps-medication-distribution"
-                            , isJust measurements.medicationDistribution
-                            )
+                            "next-steps-treatment"
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/PrenatalActivity/View.elm:1502]: @anvmn Please confirm this next step is correct, it doesn't match with the previous one.

**Review by mariano-dagostino — COMMENTED** [review]
@anvmn Looks good, with a minor question.

File: client/src/elm/Pages/PrenatalActivity/View.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Pages/PrenatalActivity/View.elm:1502]: @mariano-dagostino 

This is correct. What we define here is the icon that will be presented for activity, and these 2 share same icon.

**Pull request merged**